src/client/templates/cardModal.html

```diff
@@ -87,6 +87,7 @@ <h5 class="input-group-addon text-success hidden-xs" for="Comment">Leave A Comme
                             <span>
                                 <i class="fa fa-cog"></i>&nbsp;</span>Edit Card
                         </button>
+                        <button type="button" class="btn btn-flat text-red" data-dismiss="modal">Go Back</button>
```

Comment:
not sure this button is needed if we have one at the top and the user can hit the 'esc' key